### PR TITLE
WCM-229 Fix create_groups management command

### PR DESCRIPTION
**Pull request by SamDudley**

### CODE REVIEW

File: content/management/commands/create_groups.py [hunk 1]
```diff
@@ -174,14 +166,31 @@ def handle(self, *args, **options):
...
-        #     viewers,
-        # ]:
-        #     GroupPagePermission.objects.get_or_create(
-        #         group=group,
-        #         page=home,
-        #         permission_type="edit",
-        #     )
+        # FIXME: remove
+        # news_permissions = [
+        #     # FIXME: do we need to add newscategory still?
```
Comment (SamDudley) [inline content/management/commands/create_groups.py:171]: Thoughts?

Comment (rossmiller) [reply content/management/commands/create_groups.py:171]: Yes, news categories should be added.

**Review by rossmiller — CHANGES_REQUESTED** [review]
I've had a chat with the news team and they want to remove the concept of a news editor (they only need moderator).

**Review by rossmiller — APPROVED** [review]
LGTM!

**Pull request merged**